entity-framework/sqlite/index.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Microsoft.Data.Sqlite
+author: bricelam
+ms.date: 11/12/2019
+---
+# Microsoft.Data.Sqlite
+
+<!-- TODO -->
+
+## See also
+
+* [SQL Syntax <span class="docon docon-navigate-external" aria-hidden="true" />](https://www.sqlite.org/lang.html)
```

Comment:
@smitpatel @mairaw Is there any way to use HTML (or get it to render this) in toc.yml?
![image](https://user-images.githubusercontent.com/1226749/68718003-f6903f00-055c-11ea-9edb-a84d0e4e1cc2.png)
I hate the '⤤' we currently use.

Reply:
Submitted #1916 to iterate on this